backend/isoko-backend/src/handlers/business/post-list-business.js

```diff
@@ -9,10 +25,72 @@ exports.postListBusinessHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const requestBody = event.body && JSON.parse(event.body);
+   
+   // add defaultVal param for attributes that are optional/don't appear on non-owner list a business page
+   const name = _.get(requestBody, 'name');
+   const city = _.get(requestBody, 'city', "");
+   const state = _.get(requestBody, 'state'); 
+   const street = _.get(requestBody, 'street'); 
+   const zip = _.get(requestBody, 'zip'); 
+   const type = _.get(requestBody, 'type');
+   const tags = _.get(requestBody, 'tags');
+   const keywords = _.get(requestBody, 'keywords');
+   const shortDesc = _.get(requestBody, 'shortDesc', "");
+   const hours = _.get(requestBody, 'hours', {});
+   const links = _.get(requestBody, 'links', {});
+   const aboutOwner = _.get(requestBody, 'aboutOwner');
+   const owner = _.get(aboutOwner, 'owner', "");
+   const ownerName = _.get(aboutOwner, 'ownerName');  
+   const ownerPhone = _.get(aboutOwner, 'ownerPhone');
+   const ownerDesc = _.get(aboutOwner, 'ownerDesc', "");
+   const photo = _.get(aboutOwner, 'photo', "");
+   const lister = _.get(requestBody, 'lister');
+   
+
+   const params = {
+      TableName: BUSINESS_TABLE,
+      Item: {
+         "pk": hash(name, street), 
+         "sk": "INFO", 
+         "name": name, 
+         "city": city,
+         "street": street, 
+         "state": state, 
+         "zip": zip, 
+         "type": type, 
+         "tags": tags, 
+         "keywords": keywords,
+         "shortDesc": shortDesc, 
+         "businessId": hash(name, street), 
+         "hours": hours, 
+         "links": links, 
+         "aboutOwner": {
+            "owner": owner,
+            "ownerName": ownerName, 
+            "ownerPhone": ownerPhone, 
+            "ownerDesc": ownerDesc,
+            "photo": photo, 
+         }, 
+         "lister": lister
+      }
+   }
+   
+   const dynamoResult = await docClient
```

Comment:
I'm not sure if this applies to other HTTP methods but I think for consistency, we should wrap the put call in a try catch and return a 400 if it fails, 200 if it succeeds. You can look at my PR and copy the structure for it.